### PR TITLE
Add new post: brave vs chrome (with adblockers)

### PR DESCRIPTION
**Pull request by tomkadwill**

### CODE REVIEW

File: .well-known/brave-rewards-verification.txt [hunk 1]
```diff
@@ -1,4 +0,0 @@
-This is a Brave Rewards publisher verification file.
-
-Domain: tomkadwill.com
-Token: c94e7c6a2ccab74254ede36ff216178af22aa1534dddab838df67c7f364ab780
```
Comment (MikeRogers0) [inline .well-known/brave-rewards-verification.txt]: Hah! What the rewards program like?

File: _posts/2019-03-17-importance-of-using-linters-and-metrics-in-ruby-copy.markdown [hunk 2]
```diff
@@ -0,0 +1,122 @@
+---
+layout:     post
+title:      "Brave vs Google Chrome (with ad blockers)"
+date:       2019-03-09 16:39:00 +0000
+permalink:  'brave-vs-chrome-with-ad-blockers'
+---
+
+[Brave Browser](https://brave.com/) has been around for a few years. It became more usable when the company released a new version that was built on Chromium. A few people recommended that I switch to Brave. I tried but never stuck with it. However, with growing controversy about Google's data collection practises and Chrome's dominance, I decided to test Brave and figure out whether it's worth switching to. If Brave's browsing experiance is good, I plan to switch perminantly.
```
Comment (MikeRogers0) [inline _posts/2019-03-17-importance-of-using-linters-and-metrics-in-ruby-copy.markdown]:
```suggestion
[Brave Browser](https://brave.com/) has been around for a few years. It became more usable when the company released a new version that was built on Chromium. A few people recommended that I switch to Brave. I tried but never stuck with it. However, with growing controversy about Google's data collection practises and Chrome's dominance, I decided to test Brave and figure out whether it's worth switching to. If Brave's browsing experience is good, I plan to switch permanently.
```

File: _posts/2019-03-17-importance-of-using-linters-and-metrics-in-ruby-copy.markdown [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+### Why would anyone switch from Chrome to Brave?
+Chrome has a huge market share, [somewhere between 60-70%](https://en.wikipedia.org/wiki/Usage_share_of_web_browsers). [Many](http://www.zeldman.com/2018/12/07/browser-diversity-starts-with-us/) [people](https://redalemeden.com/blog/2019/we-need-chrome-no-more) [believe](https://www.chriskrycho.com/2017/chrome-is-not-the-standard.html) this is having a adverse affect on the web because it gives Google huge power over web standards. As a private company, they’re incentivised to use their power for personal benefit over what is good for the web.
+
+Google has come under fire lately for privacy concerns. For example, Chrome shipped a change that when you sign into any Google service, [Chrome automatically signs you in to Chrome](https://blog.cryptographyengineering.com/2018/09/23/why-im-leaving-chrome/). Then there's the concern about sharing your browsing data with a company that also has your Google search data, Andriod mobile data, YouTube browsing history, reads all your Gmail messages, can see all your trips on Google Maps. And these are just the well known Google services.
```
Comment (MikeRogers0) [inline _posts/2019-03-17-importance-of-using-linters-and-metrics-in-ruby-copy.markdown]:
```suggestion
Google has come under fire lately for privacy concerns. For example, Chrome shipped a change that when you sign into any Google service, [Chrome automatically signs you in to Google](https://blog.cryptographyengineering.com/2018/09/23/why-im-leaving-chrome/). Then there's the concern about sharing your browsing data with a company that also has your Google search data, Andriod mobile data, YouTube browsing history, reads all your Gmail messages, can see all your trips on Google Maps. And these are just the well known Google services.
```

File: _posts/2019-03-17-importance-of-using-linters-and-metrics-in-ruby-copy.markdown [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+With a fast internet connection, 20-30% speed improvement is not particularly noticeable. But for users with slower connections the difference could be noticeable.
+
+**Memory consumption**
+
+I tested the memory consumption of Brave, Chrome and Chromium. I tested them with 0 tabs, 1 tab, 2 tabs, 6 tabs, 10 tabs and 20 tabs. The tests were conducted 3 times each.
+
+![brave-chrome-chromium-memory-consumption](/assets/brave-chrome-chromium-memory-consumption.png)
+
+Chrome and Brave have the same resting (0 tabs) memory usage. As the number of tabs increase, Brave outperforms. Above 6 tabs Brave uses 20-30% less memory. I haven't found any evidence to explain why Brave has better memory usage, given that both browsers are based off Chrome. I can only guess that Chrome has more bloatware.
```
Comment (MikeRogers0) [inline _posts/2019-03-17-importance-of-using-linters-and-metrics-in-ruby-copy.markdown]: > I can only guess that Chrome has more bloatware.

It's extensions and having more stuff blocked by default which reduces the memory usage.

**Comment by MikeRogers0** [comment]
FYI: Compress your images! Current page is like 8mb. I normally use https://imageoptim.com/mac to sizes down a little, plus I avoid using that drop shadow effect on screenshots.

File: .well-known/brave-rewards-verification.txt (hunk 1 above)
Comment (tomkadwill) [reply .well-known/brave-rewards-verification.txt]: Didn't get it working 😄. I linked my YouTube account but haven't earned anything

File: _posts/2019-03-17-importance-of-using-linters-and-metrics-in-ruby-copy.markdown (hunk 2 above)
Comment (tomkadwill) [reply _posts/2019-03-17-importance-of-using-linters-and-metrics-in-ruby-copy.markdown]: 👍

File: _posts/2019-03-17-importance-of-using-linters-and-metrics-in-ruby-copy.markdown (hunk 3 above)
Comment (tomkadwill) [reply _posts/2019-03-17-importance-of-using-linters-and-metrics-in-ruby-copy.markdown]: What the diff here?

File: _posts/2019-03-17-importance-of-using-linters-and-metrics-in-ruby-copy.markdown (hunk 4 above)
Comment (tomkadwill) [reply _posts/2019-03-17-importance-of-using-linters-and-metrics-in-ruby-copy.markdown]: Do you mean Chrome has more extension installed by default?

**Comment by tomkadwill** [comment]
@MikeRogers0 👍 yeah I just optimised the images. Cut them down by about 80%

File: _posts/2019-03-17-importance-of-using-linters-and-metrics-in-ruby-copy.markdown (hunk 4 above)
Comment (MikeRogers0) [reply _posts/2019-03-17-importance-of-using-linters-and-metrics-in-ruby-copy.markdown]: @tomkadwill No, people often just install lots of rubbish ones, which try to do a lot of each page load.

**Pull request merged**